### PR TITLE
[sdesk-493] Ingest published content from remote instance of superdesk

### PR DESCRIPTION
**Pull request by marwoodandrew**

### CODE REVIEW

File: server/aap/commands/remote_sync.py [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+    def _get_remote_published_items(self):
+        """
+        Query the remote instance of superdesk for published items and process each of them
+        :return:
+        """
+        try:
+            from_count = 0
+            while True:
+                # The query excludes spiked items and returns only text items that are the last published version
+                query = {"query": {"filtered": {"filter": {"and": [{"not": {"term": {"state": "spiked"}}}, {"not": {
```
Comment (mdhaman) [inline server/aap/commands/remote_sync.py]: no need to spiked state for published repo

File: server/aap/commands/remote_sync.py [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+                    "and": [{"term": {"_type": "published"}}, {"term": {"package_type": "takes"}},
+                            {"term": {"last_published_version": False}}]}}]}}},
+                         "sort": [{"versioncreated": "asc"}],
+                         "post_filter": {"and": [{"terms": {"type": ["text"]}}]},
```
Comment (akintolga) [inline server/aap/commands/remote_sync.py]: We were using post_filter in search endpoint to get aggregations not affected by the search parameters. If search endpoint allows not using that and we don't need aggregations maybe we don't need to use it.

**Review by akintolga — APPROVED** [review]

File: server/aap/commands/remote_sync.py (hunk 2 above)
Comment (akintolga) [reply server/aap/commands/remote_sync.py]: A minor note: Search endpoint filters invisible stages but probably this wouldn't be a problem.

File: server/aap/commands/remote_sync.py (hunk 1 above)
Comment (marwoodandrew) [reply server/aap/commands/remote_sync.py]: True

**Review by mdhaman — APPROVED** [review]

**Pull request merged**